azure-pipelines.yml

```diff
@@ -60,6 +60,15 @@ stages:
       inputs:
         forceReinstallCredentialProvider: true
 
+    - template: jenkins-codesign/signType.v1.yml@templates
+
+    - task: MicroBuildSigningPlugin@2
+      inputs:
+        signType: "$(Xamarin.SignType)"
+        feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+      env:
+        TeamName: "VSCX Platform Tools"
```

Comment:
Recommend making this more configurable by applying the setting to a TeamName variable and using the variable "$(TeamName)" here instead. Helps keep the configuration as a set of variables in one place at the top of the file

Reply:
Fixed in 87c8ce6